AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -99,7 +99,7 @@ class BlackboxAssetTest:
                             builder_guid=b"bd8bf65894854fe3830e8ec3a23c35f3",
                             status=4,
                             error_count=0,
-                            warning_count=9,
+                            warning_count=22,
```

Comment:
that's a lot of warnings, we should probably catalogue all of these and figure out if it's a data change we need to make, or something else. Hopefully we can get these warning counts to start going down instead of up.